types/json-to-ast/tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
```

Comment:
new definitions cannot have anything in tslint file except extends clause. Please remove the rules here and fix all the errors.

Reply:
(from&nbsp;https://github.com/DefinitelyTyped/DefinitelyTyped/pull/35818#discussion_r288764061):

@sandersn has&nbsp;allowed me&nbsp;to&nbsp;set&nbsp;`indent` in&nbsp;`dtslint.json`:&nbsp;https://github.com/DefinitelyTyped/DefinitelyTyped/pull/33478#issuecomment-470728638.